tldw_Server_API/tests/External_Sources/test_connectors_endpoints_api.py

```
@@ -543,6 +572,352 @@ async def _fake_create_account(db, user_id, provider, display_name, email, token
+                [
+                    {
+                        "provider": "zotero",
+                        "collection_key": "COLL1234",
+                        "collection_name": "Language Models",
+                        "source_url": "https://www.zotero.org/users/123456/collections/COLL1234",
+                    }
+                ],
+                "next-zotero-cursor",
+            )
+
+    async def _fake_get_account_tokens(db, user_id, account_id):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_fake_get_account_tokens`

(ANN202)